bin/crew

```diff
@@ -421,7 +439,12 @@ def upgrade(*pkgs, build_from_source: false)
   end
 
   puts 'Packages have been updated.'.lightgreen unless rerun_upgrade
-  puts "Ruby was upgraded. Please run 'crew upgrade' again to make sure upgrades are complete.".lightblue if rerun_upgrade
+  if rerun_upgrade
+    at_exit do
+      puts "Ruby was upgraded. Rerunning 'crew upgrade' to make sure upgrades are complete.".lightblue
+      exec 'crew upgrade'
+    end
```

Comment:
Can we move line 441 to this conditional so it makes more logical sense?
```
else
  puts 'Packages have been updated.'.lightgreen
end
```

Reply:
Done.